From 22aef6c9882ac60bcf3df98e67c4b9b729a90707 Mon Sep 17 00:00:00 2001
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 16 Jun 2010 05:17:37 -0300
Subject: [PATCH 3/5] monitor: Remove pci_add command for Red Hat Enterprise Linux

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <d6c9423be5e108ae2d9f48b32ccdac550e2f91e3.1276665204.git.amit.shah@redhat.com>
Patchwork-id: 9927
O-Subject: [RHEL6 PATCH v2 1/3] monitor: Remove pci_add command for Red Hat
	Enterprise Linux
Bugzilla: 602590
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

pci_add is the old way of hot-plugging pci (nic or storage) devices.
The newer device_add is the recommended and suggested command.

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index 7044c0b..0f477da 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -851,7 +851,7 @@ STEXI
 Add drive to PCI storage controller.
 ETEXI
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
     {
         .name       = "pci_add",
         .args_type  = "pci_addr:s,type:s,opts:s?",
-- 
1.7.0.3

